src/Theme_Command.php

```diff
@@ -934,7 +864,7 @@ public function list_( $_, $assoc_args ) {
 	 * @param bool $all All flag.
 	 * @return array Same as $args if not all, otherwise all slugs.
 	 */
-	private function check_optional_args_and_all( $args, $all ) {
+	private function check_optional_args_and_all( $args, $all, $verb = 'install' ) {
```

Comment:
Isn't this covered by `ParseThemeNameInput` now?

Reply:
Yes, it is, I forgot to remove this one.
Also, the current code structure is awful and needs to be refactored, but I'll keep that for a later date.

Reply:
Resolved via https://github.com/wp-cli/extension-command/pull/259/commits/b3cf3b6b05419f9645fb8574ae34edf14b67dc48